README.md

```diff
@@ -63,22 +63,22 @@ Title:    好冷  逢考必过
  741.70 KiB / 2.63 MiB [=========>--------------------------]  27.49% 1.98 MiB/s
 ```
 
-You can also use the `-i` option to view video information, skip download.
+The `-i` option displays video information without downloading.
 
-> Note: if you have special characters in your URL, we recommend putting URL in quotation marks. (thanks @tonyxyl for pointing this out)
+> Note: wrap the URL in quotation marks if it contains special characters. (thanks @tonyxyl for pointing this out)
 > 
 > `$ annie 'https://...'`
 
-Annie does not support selecting specific video format to download, Annie will download the highest quality video that can be obtained.
+Annie does not support selecting specific video format to download. Annie will download the highest quality video available.
 
 ### Download anything else
 
-If you already got the URL of the exact resource you want, you can download it directly:
+If Annie is provided the URL of a specific resource, it will be downloaded directly:
 
 ```console
 $ annie https://img9.bcyimg.com/drawer/15294/post/1799t/1f5a87801a0711e898b12b640777720f.jpg
 
-annie doesn't support this URL by now, but it will try to download it directly
```

Comment:
sorry for this, it can't be changed here, the output comes from Annie inside

Reply:
Fixed, sorry for the number of commits, couldn't see I got it wrong on the GitHub web editing tool.